File: atelier_couture/src/data/entites/ArticleConfection.java
package data.entites;

public class ArticleConfection extends Article{

    private int prixAchat;
    private int qteAchat;
    private int montStock;
    public ArticleConfection() {}


    public int getPrixAchat() {
        return prixAchat;
    }

    public void setPrixAchat(int prixAchat) {
        this.prixAchat = prixAchat;
    }

    public int getQteAchat() {
        return qteAchat;
    }

    public void setQteAchat(int qteAchat) {
        this.qteAchat = qteAchat;
    }

    public int getMontStock() {
        return montStock;
    }

    public void setMontStock(int montStock) {
        this.montStock = montStock;
    }

    @Override
    public String toString() {
        return "ArticleConfection{" +
                "id=" + id +
                ", categorie=" + categorie +
                ", photo='" + photo + '\'' +
                ", qteStock=" + qteStock +
                ", libelle='" + libelle + '\'' +
                ", montStock=" + montStock +
                ", qteAchat=" + qteAchat +
                ", prixAchat=" + prixAchat +
                '}';
    }
}

File: model/ArticleConfection.php
<?php
require_once "./../model/Article.php";
class ArticleConfection extends Article {
    private int $prixAchat;
    private int $qteAchat;
    private int $montStock;


    public function getPrixAchat()
    {
        return $this->prixAchat;
    }


    public function setPrixAchat($prixAchat)
    {
        $this->prixAchat = $prixAchat;

        return $this;
    }


    public function getQteAchat()
    {
        return $this->qteAchat;
    }


    public function setQteAchat($qteAchat)
    {
        $this->qteAchat = $qteAchat;

        return $this;
    }


    public function getMontStock()
    {
        return $this->montStock;
    }


    public function setMontStock($montStock)
    {
        $this->montStock = $montStock;

        return $this;
    }

    public static function toArticleConfection($row):ArticleConfection{
        $articleConfection=new ArticleConfection();
        $articleConfection->setId($row['id']);
        $articleConfection->setLibelle($row['libelle']);
        $articleConfection->setPrixAchat($row['prixAchat']);
        $articleConfection->setQteStock($row['qteStock']);
        $articleConfection->setMontStock($row['montStock']);
        $articleConfection->setQteAchat($row['qteAchat']);
        $articleConfection->setCategorie($row['categorie']);
        return $articleConfection;

    }


}